proposals/lip-0070.md

```diff
@@ -525,7 +525,8 @@ This LIP adds extra functionality (commands, events, protocol logic) to the PoS
 
 ## Reference Implementation
 
-TBA
+- [COMMISSION_INCREASE_PERIOD](https://github.com/LiskHQ/lisk-sdk/commit/12c5d4c37e4d4ee46e2d7e977b29a8eda8e63e03)
```

Comment:
Maybe a link to the module itself (https://github.com/LiskHQ/lisk-sdk/tree/development/framework/src/modules/pos) would be more clear, since the user will not have to dig through many files.